picard/script.py

```diff
@@ -486,6 +486,18 @@ def func_unset(parser, name):
     return ""
 
 
+def func_delete(parser, name):
+    """
+    Deletes the variable ``name``.
+    This will unset the tag with the given name and also mark the tag for
+    deletion on save.
+    """
+    if name.startswith("_"):
```

Comment:
These 2 lines repeat all over this file. We could possibly modify it before calling all the functions?

Reply:
@samj1912 Agreed, I added https://tickets.metabrainz.org/browse/PICARD-1347